src/transactions/test/ClaimableBalanceTests.cpp

```diff
@@ -18,6 +18,43 @@
 using namespace stellar;
 using namespace stellar::txtest;
 
+#include "transactions/SignatureUtils.h"
```

Comment:
Nitpick: Weird include placement. Same thing applies to `TransactionUtils.h` actually.

Reply:
Fixed. I made the incorrect assumption `make format` would fix this.